### PR TITLE
feat: add support for voice messages

### PR DESCRIPTION
**Pull request by Mateo-tem**
**Please describe the changes this PR makes and why it should be merged:**

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**

- https://github.com/discord/discord-api-docs/pull/6082

### CODE REVIEW

File: deno/payloads/common.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const PermissionFlagsBits = {
 	 * @unstable This permission flag is currently not documented by Discord but has a known value which we will try to keep up to date.
 	 */
 	UseExternalSounds: 1n << 45n,
+	SendVoiceMessages: 1n << 46n,
```
Comment (Jiralite) [inline deno/payloads/common.ts:62]: Why is this not documented?

Comment (Mateo-tem) [reply deno/payloads/common.ts:62]: What do you mean?

Comment (Jiralite) [reply deno/payloads/common.ts:62]: Ah, don't worry. I just thought these would be documented, but none of them are.

Comment (Mateo-tem) [reply deno/payloads/common.ts:62]: Should they be documented at some point?

Comment (Jiralite) [reply deno/payloads/common.ts:62]: Maybe a simple description could be added for them in the future, but that's not a concern for this pull request (out of scope). This is fine as-is.

File: deno/payloads/v10/channel.ts [hunk 2]
```diff
@@ -1321,6 +1321,14 @@ export interface APIAttachment {
 	 * Whether this attachment is ephemeral
 	 */
 	ephemeral?: boolean;
+	/**
+	 * The duration of the audio file (currently for voice messages)
+	 */
+	duration_seconds?: number;
```
Comment (advaith1) [inline deno/payloads/v10/channel.ts:1327]:
```suggestion
	duration_secs?: number;
```

File: deno/payloads/v9/channel.ts [hunk 3]
```diff
@@ -1289,6 +1289,14 @@ export interface APIAttachment {
 	 * Whether this attachment is ephemeral
 	 */
 	ephemeral?: boolean;
+	/**
+	 * The duration of the audio file (currently for voice messages)
+	 */
+	duration_seconds?: number;
```
Comment (advaith1) [inline deno/payloads/v9/channel.ts:1295]:
```suggestion
	duration_secs?: number;
```

File: payloads/v10/channel.ts [hunk 4]
```diff
@@ -1321,6 +1321,14 @@ export interface APIAttachment {
 	 * Whether this attachment is ephemeral
 	 */
 	ephemeral?: boolean;
+	/**
+	 * The duration of the audio file (currently for voice messages)
+	 */
+	duration_seconds?: number;
```
Comment (advaith1) [inline payloads/v10/channel.ts:1327]:
```suggestion
	duration_secs?: number;
```

File: payloads/v9/channel.ts [hunk 5]
```diff
@@ -1289,6 +1289,14 @@ export interface APIAttachment {
 	 * Whether this attachment is ephemeral
 	 */
 	ephemeral?: boolean;
+	/**
+	 * The duration of the audio file (currently for voice messages)
+	 */
+	duration_seconds?: number;
```
Comment (advaith1) [inline payloads/v9/channel.ts:1295]:
```suggestion
	duration_secs?: number;
```

**Review by advaith1 — CHANGES_REQUESTED** [review]
the field is `duration_secs` not `duration_seconds`

**Comment by vladfrangu** [comment]
This PR needs an update with recent additions from discord

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**